docker-compose.yaml

```diff
@@ -31,6 +31,7 @@ services:
       POSTGRES_PASSWORD: ${POSTGRES_SCHEMA_PASSWORD:-ppp}
       OVERNIGHT_JOB_CRONTAB: ${OVERNIGHT_JOB_CRONTAB:-5 4 * * *}
       OVERNIGHT_EXPORT_BATCH_SIZE: ${OVERNIGHT_EXPORT_BATCH_SIZE:-2000}
+      PARANOID_RETENTION_PERIOD: ${PARANOID_RETENTION_PERIOD:-30}
```

Comment:
I would set this at 90 for consistency with the other settings of the same value

Reply:
oops, yeah, that was typing error